packages/nextjs/components/scaffold-stark/Balance.tsx

```diff
@@ -21,6 +22,9 @@ export const Balance = ({ address, className = "", usdMode }: BalanceProps) => {
   const { formatted, isLoading, isError } = useScaffoldEthBalance({
     address,
   });
+  const strkBalance = useScaffoldStrkBalance({
```

Comment:
can we follow the same pattern as before , so the code looks cleaner. 

```typescript
const { formatted, isLoading, isError } = useScaffoldEthBalance()
```

we can use different names like `EthBalanceFormatted`,`StrkBalanceFormatted`.